.travis.yml

```diff
@@ -15,29 +15,7 @@ install:
 - git reset --hard
 
 script:
-- set -e
-- make generate
-- if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run
-  `make generate`. Please run it and commit the changes"; git status --porcelain; false; fi
-- if diff <(git grep -c '') <(git grep -cI '') | egrep -v -e 'docs/.*\.png|swagger-ui' -e 'vendor/*' -e 'assets/*'
-  | grep '^<'; then echo "Binary files are present in git repostory."; false; fi
-- make
-- if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run
-  `make`. Please run it and commit the changes"; git status --porcelain; false; fi
-- make build-verify # verify that we set version on the packages built by bazel
-# The make bazel-test might take longer then the current timeout for a command in Travis-CI of 10 min, so adding a keep alive loop while it runs
-- |
```

Comment:
I do not understand what this one (`|`) does here (or how it works, maybe some travis-ci special char?).

Reply:
https://stackoverflow.com/questions/15540635/what-is-the-use-of-the-pipe-symbol-in-yaml/18708156
I believe this is the meaning

Reply:
Thanks.